tests/compute/cbuffer-legalize.slang.expected.txt

```diff
@@ -0,0 +1,4 @@
+3F800000
```

Comment:
Why isn’t the expected result 2 3 4 5?

Reply:
Hmm.. good point. I generated the expected output by first running the test case through `render-test` using our existing AST-to-AST path, so I relied on that code producing the right result.

I'm betting that something is going wrong and `render-test` is not setting up the constant buffer contents the way I'm expecting it to.

I'll debug it. Good catch again.

Reply:
I've found at least one issue in `render-test`, but just fixing this one hasn't changed the test output. When I look at the debug log for the test I see:

```
D3D11 ERROR: ID3D11Device::CreateBuffer: Buffer with D3D11_BIND_CONSTANT_BUFFER cannot be combined with any other bind flags.
```

The code was trying to allocate the buffer with both the `D3D11_BIND_CONSTANT_BUFFER` and `...SHADER_RESOURCE` flags.

Reply:
And the other issue seems to be that there was no logic in place to actually bind the user-specified constant buffer(s).